src/reactviews/pages/UserSurvey/userSurveyPage.tsx

```diff
@@ -150,6 +154,20 @@ export const UserSurveyPage = () => {
                             locConstants.common.cancel}
                     </Button>
                 </div>
+            </div>
+            <div className={classes.privacyDisclaimer}>
+                <Popover inline openOnHover positioning={{ coverTarget: true }}>
+                    <PopoverTrigger>
+                        <p>
+                            {locConstants.userFeedback.feedbackStatementShort}
```

Comment:
Is there a reason we need this shortened statement displayed? It doesn't seem that obvious that the extended message can be viewed by clicking the privacy statement link, or that they're even related in the first place

Reply:
Hovering with the mouse anywhere over this text:
![image](https://github.com/user-attachments/assets/da74a875-ae0d-40cf-9632-d0adb2f20d50)

Triggers the popup: 
![image](https://github.com/user-attachments/assets/23ce4ec3-bd2c-4253-b6c7-1c2e93487274)

Reply:
Oh I see, thanks for clearing that up